src/Products/ZenUI3/browser/resources/js/zenoss/zenoss.js

```diff
@@ -203,6 +203,8 @@ Zenoss.env.textMasks = {
         allowedNameTextFeedbackDash: 'Only letters, numbers, underscores, dashes and spaces allowed',
         allowedNameTextDashDot: /^[\w\s\-\.]+$/,
         allowedNameTextFeedbackDashDot: 'Only letters, numbers, underscores, dashes, periods, and spaces allowed',
+        allowedNameTextDashDotPipeComma: /^[\w\s\-\.\|\,]+$/,
+        allowedNameTextFeedbackDashDotPipeComma: 'Only letters, numbers, underscores, dashes, periods, pipes, commas, and spaces allowed',
```

Comment:
pipes (`|`) are not considered part of the set of safe URL characters.  Is there a reason why they're included here?